src/main/java/org/broadinstitute/hellbender/engine/dataflow/ReadsSource.java

```diff
@@ -87,14 +88,14 @@ public SAMFileHeader getHeader() {
      * @param intervals a list of SimpleIntervals.  These must be non-overlapping intervals or the results are undefined.
      * @return a PCollection containing all the reads that overlap the given intervals.
      */
-    public PCollection<Read> getReadPCollection(List<SimpleInterval> intervals) {
+    public PCollection<Read> getReadPCollection(List<SimpleInterval> intervals, ValidationStringency stringency) {
```

Comment:
Add an overload for `getReadPCollection()` that takes only a `List<SimpleInterval>` and calls this two-argument version, passing in `ValidationStringency.SILENT` for the second argument.